src/poliastro/twobody/orbit.py

```diff
@@ -346,7 +346,8 @@ def _sample(self, time_values, method=mean_motion):
 
     def _generate_time_values(self, nu_vals):
         # Subtract current anomaly to start from the desired point
-        M_vals = nu_to_M(nu_vals, self.ecc) - nu_to_M(self.nu, self.ecc)
+        M_vals = np.array([nu_to_M(nu_val, self.ecc).value -
```

Comment:
This `for` loop in a list comprehension might be a bit slow, wouldn't it be better to use the `@vectorize` decorator in the corresponding angles functions instead? https://numba.pydata.org/numba-doc/0.39.0/user/vectorize.html#the-vectorize-decorator (https://numba.pydata.org/numba-doc/0.39.0/reference/jit-compilation.html#numba.vectorize)